tests/unit/task_system_test.c

```diff
@@ -692,6 +692,9 @@ exec_one_of_n_threads(void *arg)
 {
    thread_config *thread_cfg = (thread_config *)arg;
 
+   // Before registration, thread ID should be in an uninit'ed state
+   ASSERT_EQUAL(INVALID_TID, platform_get_tid());
```

Comment:
The actual point of the original issue #499 was the abuse of potentially uninitialized return from `platform_get_tid()`.

Previous clean-up of `task.c` has tightened this up by always initializing the global variable, as seen in `platform.c`:

```
 10 __thread threadid xxxtid = INVALID_TID;
```

All this line is doing here is leveraging an existing test case to validate that the return from `platform_get_tid()` is no longer an uninitialized variable.